src/main/java/nextstep/step2/domain/Line.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+
+    private final List<Boolean> points = new ArrayList<>();
+    private static final float PROBABILITY = 0.4f;
+
+    public Line(int personCount) {
+        generateLine(personCount);
+    }
+
+    private void generateLine(int personCount) {
+        boolean previousValue = false;
+        for (int i = 0; i < personCount-1; i++) {
+            previousValue = generatePoint(previousValue);
+        }
+    }
+
+    private boolean generatePoint(boolean previousValue) {
+        if(previousValue) {
+            previousValue = false;
+            points.add(previousValue);
+            return previousValue;
+        }
+        previousValue = Math.random() >= PROBABILITY;
```

Comment:
```suggestion
        previousValue = 메소드_도출();
```

`Math`에 대한 접근을 우선적으로 메소드로 도출 해보아도 좋을것 같아요!